recipes/orange-widget-base/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "orange-widget-base" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "0f080a410f6aa9c929d8f7f22b7157ac3d53ca0d34de10026dedaaedad46b8e9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

As it's only used in one place, putting the hash here makes things easier for the update bot:
```suggestion
  sha256: 0f080a410f6aa9c929d8f7f22b7157ac3d53ca0d34de10026dedaaedad46b8e9
```